core/src/main/java/io/grpc/transport/inprocess/InProcessTransport.java

```diff
@@ -136,19 +137,19 @@ public synchronized void shutdown() {
     if (shutdown) {
       return;
     }
-    shutdownStatus = Status.UNAVAILABLE.withDescription("transport was requested to shut down");
-    notifyShutdown();
+    shutdownStatus = Status.OK.withDescription("transport was requested to shut down");
```

Comment:
notifyShutdown() and shutdownStatus have different purposes. One comment is for notifyShutdown(), the other is for shutdownStatus.